manifests/auth_user.pp

```diff
@@ -0,0 +1,23 @@
+# Definition: st2::auth_user
+#
+#  Friendly helper to manage standalone auth for StackStorm
+#
+# Usage
+#
+#  st2::auth_user { 'jfryman':
+#    password => 'neato!',
+#  }
+define st2::auth_user(
+  $ensure   = present,
+  $password = undef,
+) {
+  include ::st2::auth::standalone
+  $_htpasswd_file = $::st2::auth::standalone::htpasswd_file
+
+  httpauth { $name:
```

Comment:
It's only crypt, yeah. I agree with your assessment re: LDAP... that's where serious security goes. If it turns out we need to modify the `httpauth` resource to also support SHA or SSHA, then we can do that. For now, it works.